packages/docusaurus-theme-common/src/utils/docsUtils.tsx

```diff
@@ -183,3 +188,32 @@ export function isActiveSidebarItem(
 
   return false;
 }
+
+export function useDocsBreadcrumbs(): BreadcrumbItem[] {
+  const sidebar = useDocsSidebar();
+  const {pathname} = useLocation();
+  const breadcrumbs: BreadcrumbItem[] = [];
+
+  function extract(items: PropSidebar) {
```

Comment:
we have a hook `useCurrentSidebarCategory` already

I'd rather create another generic one called `useCurrentSidebarItem` + `findSidebarItem` (pure fn that can be easily tested independently)

Reply:
Might need some more info here - this `extract` fn is called recursively to pull out the sidebar item for each level, but it sounds like the purpose of `useCurrentSidebarCategory` (and what you're suggesting with `useCurrentSidebarItem`) is to extract just the category/item for the current page. Is that correct?